connectors/redis/redis.go

```diff
@@ -212,7 +220,34 @@ func (c *Connector) Remove(ctx context.Context, ei *dosa.EntityInfo, keys map[st
 		return err
 	}
 
-	return c.client.Del(cacheKey)
+	err = c.client.Del(cacheKey)
+	c.logError("Remove", err)
+	return err
+}
+
+func (c *Connector) logHitRate(method string, err error) {
+	if err != nil {
+		if _, ok := err.(*dosa.ErrNotFound); ok {
+			c.incStat("miss", method)
+			return
+		}
+		c.logError(method, err)
+		return
+	}
+	c.incStat("hit", method)
+}
+
+func (c *Connector) logError(method string, err error) {
+	if err != nil {
+		c.incStat("error", method)
+	}
+}
+
+func (c *Connector) incStat(subscope, method string) {
+	if c.stats == nil {
+		return
+	}
+	c.stats.SubScope("cache").SubScope(subscope).Tagged(map[string]string{"method": method}).Counter("redis").Inc(1)
```

Comment:
`"redis"` seems like a weird name for the counter. I would actually use the variable `subscope` in it's position (and remove it's earlier usage). I'd also not call it subscope. Maybe the following:
```
func (c *Connector) incStat(action, method string) {
 	if c.stats == nil {
 		return
 	}
 	c.stats.SubScope("cache").Tagged(map[string]string{"method": method}).Counter(action).Inc(1)
}
```